dali/base/dadfs.cpp

```diff
@@ -3472,6 +3485,7 @@ protected: friend class CDistributedFilePart;
             queryAttributes().setPropInt64("@size", totalsize);
         if (useableCheckSum)
             queryAttributes().setPropInt64("@checkSum", checkSum);
+        setHistory(fdesc);
```

Comment:
Yes it means History always present from now. If it is not then I can go back the square one with reference to a non existent object.